docs/explanation/api-and-clients.md

```diff
@@ -2,7 +2,7 @@
 
 The Pebble daemon exposes an API (HTTP over a unix socket) to allow remote clients to interact with the daemon. It can start and stop services, add configuration layers to the plan, and so on.
 
-If `pebble run` is started with the `--http <address>` option, Pebble also allows access to "untrusted" HTTP endpoints using the given TCP address (see {ref}`api-access-levels` below).
+If `pebble run` is started with the `--http <address>` option, Pebble also exposes open-access HTTP endpoints using the given TCP address (see {ref}`api-access-levels` below).
```

Comment:
Can we also change the help text from `pebble run --help` from [this](https://github.com/canonical/pebble/blob/3c8d2b6affe94b5428e6e04fcd21092ed50443d7/internals/cli/cmd_run.go#L60):

```
	"--http":        `Start HTTP API listening on this address (e.g., ":4000")`,
```

to something like:

```
	"--http":        `Start HTTP API listening on this address (e.g., ":4000") and expose open-access endpoints`,
```

Reply:
Good suggestion. I've added another commit with this change